Makefile

```diff
@@ -19,8 +20,21 @@ tidy:
 deps:
 	go install github.com/mgechev/revive@latest
 	go install golang.org/x/tools/cmd/goimports@latest
+	go install github.com/swaggo/swag/cmd/swag@v1.8.0
```

Comment:
How come `go.mod` and `CHANGELOG.md` says `v1.7.9` when this is `v1.8.0`?

Reply:
Oops. Fixed.